src/pages/settings/Preferences/ThemePage.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'underscore';
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import HeaderWithBackButton from '../../../components/HeaderWithBackButton';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import SelectionListRadio from '../../../components/SelectionListRadio';
+import styles from '../../../styles/styles';
+// import * as App from '../../../libs/actions/App';
+import ONYXKEYS from '../../../ONYXKEYS';
+import CONST from '../../../CONST';
+import compose from '../../../libs/compose';
+import Text from '../../../components/Text';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** The theme of the app */
+    preferredTheme: PropTypes.string,
+};
+
+const defaultProps = {
+    preferredTheme: CONST.DEFAULT_THEME,
+};
+
+function ThemePage(props) {
+    const localesToThemes = _.map(props.translate('themePage.themes'), (theme, key) => ({
+        value: key,
+        text: theme.label,
+        keyForList: key,
+        isSelected: props.preferredTheme === key,
+    }));
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <HeaderWithBackButton
+                title={props.translate('themePage.theme')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_PREFERENCES)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+
+            <Text style={[styles.mh5, styles.mv4]}>{props.translate('themePage.chooseThemeBelowOrSync')}</Text>
+
+            <SelectionListRadio
+                sections={[{data: localesToThemes}]}
+                // onSelectRow={(theme) => User.updateTheme(theme.value)}
```

Comment:
Let's add this helper function and add this line to the [going live PR](https://github.com/Expensify/App/pull/21669/files) after this PR is merged